packages/common/src/authorization/organizationMemberAbility.ts

```diff
@@ -17,9 +17,6 @@ export const organizationMemberAbilities: Record<
     ) => void
 > = {
     member(member, { can }) {
-        can('create', 'InviteLink', {
```

Comment:
don't we need to move this `can create InviteLink` to admins ?

Reply:
I tested this and it works as expected. How is this possible if we don't have this ability anywhere ?

Reply:
Because admins have permissions to 'manage' and that includes 'create'.

``` 
can('manage', 'InviteLink', {
     organizationUuid: member.organizationUuid,
});
```